packages/components/src/components/Alert/Alert.tsx

```diff
@@ -194,6 +211,44 @@ export const Alert: FC<AlertProps> = ({
     lineHeight: 30,
   }
 
+  const _header = () => {
+    const headerText = header ? <Text style={headerFont}>{header}</Text> : null
```

Comment:
Seems like it behaves fine (in iOS at least, didn't check Android) with the screen reader, but don't like that when header is undefined (e.g. valid non-expandable version) we're returning a null wrapped in `<View accessible={true} aria-label={a11yLabel} role="heading">`.

I think these header checks are probably a byproduct of the prior description-in-header functionality.

We should probably just open this function with:
`if (!header) return null`
and clean up these conditionals a bit to be sure no awkwardness could happen around returning a valid view with no content in it when we don't have a header.

Reply:
Returning `null` if no header and cleaned up `headerText` and `a11yLabel` logic accordingly.